.prettierignore

```diff
@@ -1,13 +1,11 @@
 .DS_Store
 node_modules
 /build
-/.svelte-kit
 /package
+.docusaurus
 .env
 .env.*
 !.env.example
 
-# Ignore files for PNPM, NPM and YARN
-pnpm-lock.yaml
-package-lock.json
+# Ignore files for YARN
 yarn.lock
```

Comment:
lock files ensure my build and your build use the same dependencies in different environments. AFAIK they are usually only ignored when developing libraries. Any reason to ignore here?

Reply:
This is `.prettierignore`, not `.gitignore`. `yarn.lock` is being committed, just not linted.